bin/ncp/CONFIG/nc-passwd.sh

```diff
@@ -8,10 +8,16 @@
 # More at: https://ownyourbits.com
 #
 
-
+cleanup()
```

Comment:
let's do this for every ncp-app

Reply:
Do you think I can implement this as an opt-out instead of an opt-in (or not optional at all) without breaking things?
If so I'd include it in `library.sh/run_app_unsafe` instead

Reply:
Btw, will trap even trigger if a function returns? :thinking:

Reply:
look at my changes below, I am building them. I implemented it in `run_app_unsafe`

https://github.com/nextcloud/nextcloudpi/pull/1062#issuecomment-576725577